.github/workflows/build.yml

```diff
@@ -107,21 +107,13 @@ jobs:
       with:
         files: results.xml
 
-    - name: Build Release
-      run: |
-        cargo build --release
-        cargo install cargo-bundle
-        cargo bundle --release
+    - name: Make App
+      run: make dmg
 
-    - name: Create .dmg file
+    - name: Upload Application
       run: |
-        hdiutil create Neovide-uncompressed.dmg -volname "Neovide" -srcfolder target/release/bundle/osx
-        hdiutil convert Neovide-uncompressed.dmg -format UDZO -o Neovide.dmg
-
-    - uses: actions/upload-artifact@v1

```

Comment:
Why do you want to replace this with a script? The action works fine doesn't it? Sorry I'm out of the loop.